railgun/test/integration/ingress_bulk_test.go

```diff
@@ -0,0 +1,145 @@
+//+build integration_tests
+
+package integration
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kong/kubernetes-ingress-controller/pkg/annotations"
+	"github.com/kong/kubernetes-testing-framework/pkg/utils/kubernetes/generators"
+)
+
+const (
+	maxBatchSize             = 50
```

Comment:
can we make the size a parameter instead of hard code

Reply:
231df0c357ca2e258f9a84c43f70445cda68489e resolves this by making the value configurable.